_posts/2022-01-19-7.0.0-release.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: post
+title: "Apache Arrow 7.0.0 Release"
+date: "2022-01-19 00:00:00 -0600"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 7.0.0 release. This covers
+over 3 months of development work and includes [**474 resolved issues**][1]
+from [**?? distinct contributors**][2]. See the Install Page to learn how to
+get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 6.0.1 release, Rémi Dattai and Alessandro Molina have been invited to be committers.
+Daniël Heres and Yibo Cai have joined the Project Management Committee (PMC).
+Thanks for your contributions and participation in the project!
+
+## Columnar Format Notes
+
+TBD
+
+## Arrow Flight RPC notes
+
+TBD
+
+## C++ notes
+
+TBD
+
+### Compute Layer
+
+TBD
+
+### Dataset Layer
+
+TBD
+
+### IO and Filesystem Layer
+
+TBD
+
+### Parquet
+
+TBD
+
+## C# notes
+
+TBD
+
+## Go notes
+
+### Bug Fixes
+
+TBD
+
+### Enhancements
+
+TBD
+
+## Java notes
+
+TBD
+
+## JavaScript notes
+
+TBD
+
+## Python notes
+
+TBD
+
+## R notes
+
+TBD
+
+## Ruby and C GLib notes
+
+### Ruby
+
+The updates of Red Arrow etc. consists of the following improvements:
+
+TBD
+
+### C GLib
+
+The updates of Arrow GLib etc. consists of the following improvements:
+
+TBD
+
+## Rust notes
+
+Rust releases minor versions every 2 weeks in addition to a major
```

Comment:
FYI I added content for the 7.0.0 release of Rust -- fyi @jhorstmann @Dandandan @houqp @jorgecarleitao @andygrove @nevi-me